### ISSUE TITLE
Suggest UI refresh (better navigation)

### ISSUE THREAD

**Issue by j-levy** [R-a-dio/R-a-dio-Amazing-Android-App#21]
If more "last played" songs are added, and local favorites, and maybe parameters are also implemented, the current UI won't be practical. The more pages are added, the more we need to swipe to reach the page we want. At the same time, the sidenav with last played / queue is not useful, since these info do not need to be displayed everywhere in the app (they could be in a single page).

I would suggest to use the sidenav as navigation menu, with buttons to reach every page. This is the current design norm for most applications. At the bottom of the sidenav, a button for parameters could be placed. This way, much more pages can be added and it will only take 1 swipe + 1 tap to access it. 

Then, the page navigation can stay as it is (with small dots at the top), or the dots could be replaced by tabs (easier to select). Most, if not all applications use tabs for navigation, but we may have too many pages for tabs to be displayed properly, so keeping navigation dots would be alright.

**Comment by j-levy** [comment]
After discussion with Kethsar, we agreed on starting anew for a more modern app. I won't contribute here any more, since the objective has shifted. I started with Xamarin.Forms (C#) because it appeared like a nice compromise between final product performance (speed/size) and ease of programming. Also it allows for cross-platform development, so we could build an iOS app too that shares a lot of code with the Android version. 
The repo is here : https://github.com/j-levy/RadioApp-Xamarin
I started implementing backend features : playback and API fetches are done, and a job scheduler has been implemented for Android to ping the API periodically. (The same should be done for iOS). We will try to set up a first prototype of interface for the Now Playing screen as next objective. Help is more than welcome!

**Comment by resttime** [comment]
@j-levy Just saw this issue. Good luck, it should be somewhat straightforward to make a rewrite since I don't particularly recall anything complex outside of implementing the lifecycle and features of a regular app. Let me know if you need any help.

**Comment by yattoz** [comment]
Actually I started creating a new app about a month ago and I've been using it personally, adding features and new stuff. It's only android (no Xamarin, cause I couldn't do what I wanted with it)
See repo here : https://github.com/yattoz/Radio2
There's a cool GIF if you don't want to run it, but it should be as simple as clone, sync gradle, run. 

I reached a point of "release 1.0" (there's a branch with that name) with a decent number of features and no known "nasty" bugs. Lately I started to add new little things and some refinements for a 1.1 update (see the kanban under "Projects" ). There's not much left in what I planned, so I could make a "release1.1" branch in a few days maybe (I'm not sure how long the last feature would take.)

I told Kethsar about this, he told me he would take a look and maybe publish it one day. So you might want to discuss with him for publication on Google Play or other related stuff. 

(Oh and for clarity purposes I decided to host personal projects on a separate github account, but I'm the same guy)
